packages/withoutHTML/src/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @description Cleanups a string from HTML tags.
+ * @param {String} text
+ * @example
+ * withoutDiacritics('<b>Test</b>') // returns 'Test'
+ */
+
+function removeHTML(ogstr)
```

Comment:
This function is redundant, also I think you are not using prettier for linting and the code styling is a bit off.

Reply:
I used the GitHub web editor, so sadly no linting or anything like that. I will lint it and update the files.